### DISCUSSION TITLE
Bug is back function

### DISCUSSION THREAD

**Discussion by IWareQ** [community/community#7187] [Mobile] [answered]
https://user-images.githubusercontent.com/51165317/140029895-0fb8d021-aad9-4713-b77e-f9eef9223021.mp4

**Comment by brianlovin** [comment, marked as answer]
Thank you for the report! I can reproduce on my end, I'll flag this for our team and we'll get a fix out as soon as possible.

**Reply by okcha17** [reply]
Senks

**Comment by Vannsansan** [comment]
Wedding VS Coconut Chicken, which is more important? #Shorts | Ms Yeah / 办公室小野 Ms Yeah
https://www.youtube.com/watch?v=JIQajAla7d8

**Reply by okcha17** [reply]
Senks

**Comment by Vannsansan** [comment]
😍🤩🥰
